frontend/app/src/services/rotkehlchen-api.ts

```diff
@@ -393,18 +393,27 @@ export class RotkehlchenApi {
 
   queryTaskResult<T>(
     id: number,
-    numericKeys?: string[] | null
+    numericKeys?: string[] | null,
+    transform: boolean = true
```

Comment:
So is this because the task returns a file instead?

Reply:
This api for download json for browser, to not transform the keys to camel case This is to match with json exported for electron.
The json returned for electron is not in camel case.